release-notes/Cabal-3.14.0.0.md

```diff
@@ -74,7 +74,7 @@
     -> IO (SymbolicPath dir1 File)
   ```
 
-  See Note [Symbolic paths] in `Distribution.Utils.Path` for further information
+  See `Note [Symbolic paths]` in `Distribution.Utils.Path` for further information
```

Comment:
Not sure these backticks make sense to me: I reserve them for code. But again: no harm...

Reply:
I'll axe it

Reply:
If you go look at the original PR, we had a back-and-forth about this. (Which Mergify committed in the middle of.)